### PR TITLE
Update Azure pipelines to build from Nexus branch

### PR DESCRIPTION
**Pull request by phunkyfish**

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+          CC: gcc
+          CXX: g++
+          DEBIAN_BUILD: true
+        #- os: ubuntu-18.04
+          #CC: gcc
+          #CXX: g++
+        #- os: ubuntu-18.04
+          #CC: clang
+          #CXX: clang++
+        #- os: macos-10.15
```
Comment (ksooo) [inline .github/workflows/build.yml]: I would remove this completely.

Comment (phunkyfish) [reply .github/workflows/build.yml]: Done

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+        - name: "Debian package test"
+          os: ubuntu-20.04
```
Comment (ksooo) [inline .github/workflows/build.yml]: ubuntu-latest is better choice, with this we will not need to update this once 20.04 runner will be EOLed.

Comment (phunkyfish) [reply .github/workflows/build.yml]: I need ffmpeg 4 to be able to build this. So I can't use latest. But something is not working anyway as its using ffmpeg5 for some reason.

**Comment by phunkyfish** [comment]
So we run over the allowed 60 minute build run for azure. Not sure how to get around the timeout but its running better than before.

**Pull request merged**